Assets/Demo/DemoControls.cs

```diff
@@ -495,22 +497,27 @@ public bool Contains(InputAction action)
     {
         return asset.Contains(action);
     }
+
     public IEnumerator<InputAction> GetEnumerator()
     {
         return asset.GetEnumerator();
     }
+
     IEnumerator IEnumerable.GetEnumerator()
     {
         return GetEnumerator();
     }
+
     public void Enable()
     {
         asset.Enable();
     }
+
     public void Disable()
     {
         asset.Disable();
     }
+
```

Comment:
For generated code, I think it'd be great to generate it such that it already complies with formatting.

Reply:
Agreed - was planning to do that in a separate PR